app/services/holdings_service.py

```
@@ -0,0 +1,519 @@
+            })
+
+        return holdings
+
+    async def _get_or_update_stock_info(
+        self,
+        db: AsyncSession,
+        symbol: str,
+        name: str,
+        instrument_type: str,
+        exchange: str,
+        current_price: float = None
```

Ruff (0.14.2)

Unused method argument: `current_price`

(ARG002)